From 6da2ddcd71be9ca6def974e7614e2cc5a2bd7faf Mon Sep 17 00:00:00 2001
From: Michael Yartys <michael.yartys@protonmail.com>
Date: Thu, 20 Jan 2022 12:34:54 +0100
Subject: [PATCH 1/1] defconfig: Fedora configuration

Change defconfig in order to build for Fedora. Make sure Operating Channel
Validation (OCV) is enabled.

Signed-off by: Lubomir Rintel <lkundrak@v3.sk>
Signed-off-by: Michael Yartys <michael.yartys@protonmail.com>
---
 wpa_supplicant/defconfig | 17 ++++++++++-------
 1 file changed, 10 insertions(+), 7 deletions(-)

diff --git a/wpa_supplicant/defconfig b/wpa_supplicant/defconfig
index a4719dbb5..d53a25b3f 100644
--- a/wpa_supplicant/defconfig
+++ b/wpa_supplicant/defconfig
@@ -146,7 +146,7 @@ CONFIG_EAP_PAX=y
 CONFIG_EAP_LEAP=y
 
 # EAP-AKA (enable CONFIG_PCSC, if EAP-AKA is used)
-#CONFIG_EAP_AKA=y
+CONFIG_EAP_AKA=y
 
 # EAP-AKA' (enable CONFIG_PCSC, if EAP-AKA' is used).
 # This requires CONFIG_EAP_AKA to be enabled, too.
@@ -311,7 +311,7 @@ CONFIG_BACKEND=file
 #CONFIG_NO_LINUX_PACKET_SOCKET_WAR=y
 
 # Support Operating Channel Validation
-#CONFIG_OCV=y
+CONFIG_OCV=y
 
 # Select TLS implementation
 # openssl = OpenSSL (default)
@@ -337,7 +337,7 @@ CONFIG_BACKEND=file
 
 # Select which ciphers to use by default with OpenSSL if the user does not
 # specify them.
-#CONFIG_TLS_DEFAULT_CIPHERS="DEFAULT:!EXP:!LOW"
+CONFIG_TLS_DEFAULT_CIPHERS="PROFILE=SYSTEM:3DES"
 
 # If CONFIG_TLS=internal is used, additional library and include paths are
 # needed for LibTomMath. Alternatively, an integrated, minimal version of
@@ -469,7 +469,7 @@ CONFIG_DEBUG_SYSLOG=y
 # Should we attempt to use the getrandom(2) call that provides more reliable
 # yet secure randomness source than /dev/random on Linux 3.17 and newer.
 # Requires glibc 2.25 to build, falls back to /dev/random if unavailable.
-#CONFIG_GETRANDOM=y
+CONFIG_GETRANDOM=y
 
 # IEEE 802.11ac (Very High Throughput) support (mainly for AP mode)
 CONFIG_IEEE80211AC=y
@@ -587,7 +587,7 @@ CONFIG_IBSS_RSN=y
 #CONFIG_PMKSA_CACHE_EXTERNAL=y
 
 # Mesh Networking (IEEE 802.11s)
-#CONFIG_MESH=y
+CONFIG_MESH=y
 
 # Background scanning modules
 # These can be used to request wpa_supplicant to perform background scanning
@@ -601,7 +601,7 @@ CONFIG_BGSCAN_SIMPLE=y
 
 # Opportunistic Wireless Encryption (OWE)
 # Experimental implementation of draft-harkins-owe-07.txt
-#CONFIG_OWE=y
+CONFIG_OWE=y
 
 # Device Provisioning Protocol (DPP) (also known as Wi-Fi Easy Connect)
 CONFIG_DPP=y
@@ -617,7 +617,7 @@ CONFIG_DPP2=y
 # functionality needed to use WEP is available in the current wpa_supplicant
 # release under this optional build parameter. This functionality is subject to
 # be completely removed in a future release.
-#CONFIG_WEP=y
+CONFIG_WEP=y
 
 # Remove all TKIP functionality
 # TKIP is an old cryptographic data confidentiality algorithm that is not
@@ -633,3 +633,6 @@ CONFIG_DPP2=y
 # design is still subject to change. As such, this should not yet be enabled in
 # production use.
 #CONFIG_PASN=y
+#
+CONFIG_SUITEB192=y
+
-- 
2.34.1

